Cargo.toml

```diff
@@ -1,3 +1,8 @@
+[profile.release]
+codegen-units = 1
+lto = true
+opt-level = 3
```

Comment:
Will this have any impact on the released crates, or just the examples?

Reply:
This will impact every release not just examples. I could not just put to examples crate but the cargo requires the top. 

I will remove this and find a way to add builder flags on command line.